frontend/src/Components/ContentComponent.tsx

```diff
@@ -77,7 +77,7 @@ function processVideoEmbed(img: HTMLImageElement) {
             video.src = videoUrl;
             video.controls = true;
             video.autoplay = true;
-            video.loop = !!img.dataset.loop;
+            video.loop = !!img.dataset.loop && img.dataset.loop !== 'false';
```

Comment:
let's do `img.dataset.loop.toLowerCase()` in the comparisoin just in case it would contain `FALSE`

Reply:
It won't as it comes from here:
https://github.com/spaceshelter/orbitar/blob/e501069f350277e1bbd57939cb9f44daba32809b/backend/src/parser/TheParser.ts#L317